ui/raidboss/data/02-arr/raid/t9.js

```diff
@@ -542,7 +541,7 @@
         'Divebomb': 'ダイブボム',
         'Fireball': 'ファイアボール',
         'Ghost': 'ゴースト',
-        'Heavensfall': 'ヘヴンスフォール',
+        'Heavensfall': '天地崩壊',
```

Comment:
Was this one intentional?

Reply:
That's not what I did. I just followed last version what i didn't touched